docs/concepts/models.md

```diff
@@ -772,12 +772,6 @@ Order(id=1, product=ResponseModel[Product](content=Product(name='Apple', price=0
 """
 ```
 
-!!! tip
```

Comment:
It's probably worth noting that if you don't do this you will incur a bit of validation overhead, and re-trigger things like `@model_validator`s that will run any time the instance is validated, since the disagreement in parametrization means it won't skip validation